src/react-components/src/components/02-molecules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,80 @@
+/* MN-10: Pagination */
+
+import * as React from "react";
+import bem from "../../../utils/bem";
+import FormDropdown from "../../01-atoms/Forms/FormDropdown";
+import Label from "../../01-atoms/Forms/Label";
+import HelperErrorText from "../Helpers/HelperErrorText";
+
+export interface DropdownProps {
+  dropdownId: string;
+  isRequired: boolean;
+  labelPosition: string;
+  labelId?: string;
+  labelText?: string;
+  ariaLabel?: string;
+  helperContent?: JSX.Element;
+  helperContentId?: string;
+  hasError?: boolean;
+  errorContentId?: string;
+  errorContent?: JSX.Element;
+  dropdownOptions: string[];
+  onSelectChange: (event: React.FormEvent) => void;
+  onSelectBlur: (event: React.FormEvent) => void;
+}
+
+export default function Dropdown(props: DropdownProps) {
+  const { dropdownId, isRequired,
+    labelId, labelPosition, labelText, ariaLabel,
+    helperContent, helperContentId,
+    hasError, errorContent, errorContentId,
+    dropdownOptions,
+    onSelectChange, onSelectBlur } = props;
+
+  let modifiers = [];
+
+  const baseClass = "dropdown";
+  if (!["left", "top", "none"].includes(labelPosition)) {
```

Comment:
You'd be better off declaring the type of `labelPosition` to be of type `"left" | "top" | "none"` instead of `"string"`. Then you do not have to make this check here. This is great for Typescript but I do wonder how the compiled version of it holds up. It probably does this same error check and throw behinds the scenes for you.

You should also declare the default prop value of it to be `"none"`.

Reply:
I'd like to see this change in the Typescript code, but when it is used by others, it actually doesn't matter unless if they use Typescript in the app and use the Typescript source files. Otherwise, the compiled code doesn't really check for this and the dev could potentially use a value other than "left", "top", "none". So maybe you can include both the typescript type check as well as keeping this error check. I'll try to test it out.

Reply:
Okay, I tested this in a SimplyE typescript app and in SFR and the compiled code doesn't care about the Typescript type. So I would have both checks in place to catch compile time and run time errors in Typescript or non-Typescript projects.